config/src/config/storage_config.rs

```diff
@@ -63,8 +64,8 @@ impl Default for RocksdbConfigs {
         Self {
             ledger_db_config: RocksdbConfig::default(),
             state_merkle_db_config: RocksdbConfig::default(),
-            use_kv_db: false,
-            kv_db_config: RocksdbConfig::default(),
+            use_state_kv_db: false,
+            state_kv_db_config: RocksdbConfig::default(),
```

Comment:
love the rename, please make it a separate commit though

Reply:
Done